ext/RMagick/rmagick.h

```diff
@@ -1057,7 +1092,15 @@ extern const char *StretchType_name(StretchType);
 extern VALUE  StyleType_new(StyleType);
 extern const char *StyleType_name(StyleType);
 extern VALUE  VirtualPixelMethod_new(VirtualPixelMethod);
-
+#if defined(IMAGEMAGICK_7)
+extern VALUE  FilterType_new(FilterType);
+extern VALUE  LayerMethod_new(LayerMethod);
+extern VALUE  PixelInterpolateMethod_new(PixelInterpolateMethod);
```

Comment:
Are these methods only renamed, or do they behave differently? This seems like it might be another case where we could alias for a smooth transition.

Reply:
These are renamed and might have different values. As stated before I would like to have a different API for ImageMagick 6 and 7 so renaming this was the obvious choice for me.

Reply:
We can rename them to the ImageMagick 7 names because they are only used internally.